packages/@lwc/babel-plugin-component/src/decorators/wire/validate.ts

```diff
@@ -36,13 +41,15 @@ function validateWireParameters(path) {
         });
     }
 
+    // @ts-ignore
```

Comment:
TypeScript thinks this is a `never` here; no idea why. I figure we can fix this later.

Reply:
It's because, on line 38, the typescript compiler thinks that all member expressions will throw an error. The babel types don't capture the fact that `computed` has also to be `true`.

To avoid this, you can replace the condition line 38 with:

```ts
if (id.isMemberExpression() && id.node.computed === true) {
```

Reply:
Added a TODO.